v22.1/changefeed-sinks.md

```diff
@@ -86,6 +87,42 @@ The configurable fields include:
   * `"NONE"`: no Kafka brokers are required to acknowledge that they have committed the message. This will decrease latency and increase throughput, but comes at the cost of lower consistency.
   * `"ALL"`: a quorum must be reached (that is, most Kafka brokers have committed the message) before the leader can acknowledge. This is the highest consistency level.
 
+## Google Cloud Pub/Sub
+
+{{site.data.alerts.callout_info}}
+The Google Cloud Pub/Sub sink is currently in **beta**. For more information, read about its usage considerations, available [parameters](create-changefeed.html#parameters), and [options](create-changefeed.html#options).
+{{site.data.alerts.end}}
+
+<span class="version-tag">New in v22.1:</span> Changefeeds can deliver messages to a Google Cloud Pub/Sub sink, which is integrated with Google Cloud Platform.
+
+A Pub/Sub sink URI follows this example:
+
+~~~
+'gcpubsub://{project name}?region={region}&topic_name={topic name}&AUTH=specified&CREDENTIALS={base64-encoded key}'
+~~~
+
+URI Parameter      | Description
+-------------------+------------------------------------------------------------------
+`project name`     | The [Google Cloud Project](https://cloud.google.com/resource-manager/docs/creating-managing-projects) name.
+`region`           | The single region to which all output will be sent.
+`topic_name`       | (Optional) The topic name to which messages will be sent. See the following section on [Topic Naming](#topic-naming) for detail on how topics are created.
+`AUTH`             | The authentication parameter can define either `specified` or `implicit` authentication. Passing your [Service Account](https://cloud.google.com/iam/docs/understanding-service-accounts) credentials with the URI or implicitly in your environment respectively. [Use Cloud Storage for Bulk Operations](use-cloud-storage-for-bulk-operations.html#authentication) provides more detail on authentication to cloud storage sinks.
+`CREDENTIALS`      | (Optional) The base64-encoded credentials of your Google [Service Account](https://cloud.google.com/iam/docs/understanding-service-accounts) credentials. Include with `AUTH=specified`.
+
+When using Pub/Sub as your downstream sink, consider the following:
```

Comment:
I can add that you must specify a region. Instead of adding here that we are *planning* to add an unspecified/unordered option, what do you think about framing that within a limitation with a GH issue? We try not to add future plans in a "We're planning to add this in a future release..." to avoid either over-promising, or just general confusion on _when_ that could happen. 

Limitation could be framed as "X is not available in Pub/Sub [Tracking Issue]", which at least provides the user with context. (I can link to the limitation from this point as well.